### ISSUE TITLE
Dokku output filled with bash environment related noise

### ISSUE THREAD

**Issue by jonatkinson** [dokku/dokku#3983]
## Description of problem

Dokku output is polluted with a lot of noise related to (I think) the bash environment. This occurs both when using `dokku` commands on the host, and in the output when running `git push dokku`.

The host is running Debian Buster. Dokku itself is `0.20.4`, installed via the `wget`/`bootstrap.sh` method.

All operations do seem to complete successfully. Sample output below.

The output is less significant in some commands, for example, when using `dokku config:*` commands, nothing is seen. In others, like a `git push`, more than 70% of the output seems to be these lines.

### How reproducible

- Run any `dokku` command on the host.
- Push to the `dokku` remote.

#### Actual Results

This is a sample, but representative.

```
...
-----> Setting config vars
       DOKKU_PROXY_PORT_MAP:  http:80:5000
/tmp/bashenv.505361674: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
-----> Setting config vars
       DOKKU_PROXY_PORT_MAP:  http:80:5000 https:443:5000
/tmp/bashenv.826611486: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.453730364: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.748838143: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.542676436: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.535390724: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.765516068: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.537868931: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.181962458: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
-----> Configuring myapp.remote.jonatkinson.co.uk...(using built-in template)
-----> Configuring myapp.co...(using built-in template)
-----> Creating https nginx.conf
...
```

#### Expected Results

```
...
-----> Setting config vars
       DOKKU_PROXY_PORT_MAP:  http:80:5000
-----> Setting config vars
       DOKKU_PROXY_PORT_MAP:  http:80:5000 https:443:5000
-----> Configuring myapp.remote.jonatkinson.co.uk...(using built-in template)
-----> Configuring myapp.co...(using built-in template)
-----> Creating https nginx.conf
...
```

### `dokku report APP_NAME` output

-----> uname: Linux remote 4.19.0-8-amd64 #1 SMP Debian 4.19.98-1+deb10u1 (2020-04-27) x86_64 GNU/Linux
-----> memory: 
                     total        used        free      shared  buff/cache   available
       Mem:         128820        2279      123617           0        2923      125414
       Swap:          4095           0        4095
-----> docker version: 
       Client: Docker Engine - Community
        Version:           19.03.8
        API version:       1.40
        Go version:        go1.12.17
        Git commit:        afacb8b7f0
        Built:             Wed Mar 11 01:25:56 2020
        OS/Arch:           linux/amd64
        Experimental:      false
       
       Server: Docker Engine - Community
        Engine:
         Version:          19.03.8
         API version:      1.40 (minimum version 1.12)
         Go version:       go1.12.17
         Git commit:       afacb8b7f0
         Built:            Wed Mar 11 01:24:28 2020
         OS/Arch:          linux/amd64
         Experimental:     false
        containerd:
         Version:          1.2.13
         GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
        runc:
         Version:          1.0.0-rc10
         GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
        docker-init:
         Version:          0.18.0
         GitCommit:        fec3683
-----> docker daemon info: 
       Client:
        Debug Mode: true
       
       Server:
        Containers: 9
         Running: 1
         Paused: 0
         Stopped: 8
        Images: 12
        Server Version: 19.03.8
        Storage Driver: overlay2
         Backing Filesystem: <unknown>
         Supports d_type: true
         Native Overlay Diff: true
        Logging Driver: json-file
        Cgroup Driver: cgroupfs
        Plugins:
         Volume: local
         Network: bridge host ipvlan macvlan null overlay
         Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
        Swarm: inactive
        Runtimes: runc
        Default Runtime: runc
        Init Binary: docker-init
WARNING: No swap limit support
        containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
        runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
        init version: fec3683
        Security Options:
         apparmor
         seccomp
          Profile: default
        Kernel Version: 4.19.0-8-amd64
        Operating System: Debian GNU/Linux 10 (buster)
        OSType: linux
        Architecture: x86_64
        CPUs: 12
        Total Memory: 125.8GiB
        Name: remote
        ID: FH5Y:J4XN:WHGC:EOQT:AH5A:76FV:AGTP:MQJX:2F5W:A74K:T4IZ:VBUR
        Docker Root Dir: /var/lib/docker
        Debug Mode: false
        Registry: https://index.docker.io/v1/
        Labels:
        Experimental: false
        Insecure Registries:
         127.0.0.0/8
        Live Restore Enabled: false
       
-----> sigil version: 0.6.0
-----> herokuish version: 
       herokuish: 0.5.12
       buildpacks:
         heroku-buildpack-multi     v1.0.0
         heroku-buildpack-ruby      v215
         heroku-buildpack-nodejs    v170
         heroku-buildpack-clojure   v84
         heroku-buildpack-python    v169
         heroku-buildpack-java      v66
         heroku-buildpack-gradle    v31
         heroku-buildpack-scala     v87
         heroku-buildpack-play      v26
         heroku-buildpack-php       v173
         heroku-buildpack-go        v141
         buildpack-nginx            v13
-----> dokku version: dokku version 0.20.4
-----> dokku plugins: 
       plugn: 0.4.0
         00_dokku-standard    0.20.4 enabled    dokku core standard plugin
         20_events            0.20.4 enabled    dokku core events logging plugin
         app-json             0.20.4 enabled    dokku core app-json plugin
         apps                 0.20.4 enabled    dokku core apps plugin
         builder-dockerfile   0.20.4 enabled    dokku core builder-dockerfile plugin
         builder-herokuish    0.20.4 enabled    dokku core builder-herokuish plugin
         buildpacks           0.20.4 enabled    dokku core buildpacks plugin
         certs                0.20.4 enabled    dokku core certificate management plugin
         checks               0.20.4 enabled    dokku core checks plugin
         common               0.20.4 enabled    dokku core common plugin
         config               0.20.4 enabled    dokku core config plugin
         docker-options       0.20.4 enabled    dokku core docker-options plugin
         domains              0.20.4 enabled    dokku core domains plugin
         enter                0.20.4 enabled    dokku core enter plugin
         git                  0.20.4 enabled    dokku core git plugin
         letsencrypt          0.9.3 enabled    Automated installation of let's encrypt TLS certificates
         logs                 0.20.4 enabled    dokku core logs plugin
         network              0.20.4 enabled    dokku core network plugin
         nginx-vhosts         0.20.4 enabled    dokku core nginx-vhosts plugin
         plugin               0.20.4 enabled    dokku core plugin plugin
         proxy                0.20.4 enabled    dokku core proxy plugin
         ps                   0.20.4 enabled    dokku core ps plugin
         repo                 0.20.4 enabled    dokku core repo plugin
         resource             0.20.4 enabled    dokku core resource plugin
         scheduler-docker-local 0.20.4 enabled    dokku core scheduler-docker-local plugin
         shell                0.20.4 enabled    dokku core shell plugin
         ssh-keys             0.20.4 enabled    dokku core ssh-keys plugin
         storage              0.20.4 enabled    dokku core storage plugin
         tags                 0.20.4 enabled    dokku core tags plugin
         tar                  0.20.4 enabled    dokku core tar plugin
         trace                0.20.4 enabled    dokku core trace plugin
/tmp/bashenv.312847697: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
=====> myapp app information
       App deploy source:             
       App dir:                       /home/dokku/myapp
       App locked:                    false
=====> myapp buildpacks information
       Buildpacks list:               
CN = myapp.remote.jonatkinson.co.uk
error 20 at 0 depth lookup: unable to get local issuer certificate
=====> myapp ssl information
       Ssl dir:                       /home/dokku/myapp/tls
       Ssl enabled:                   true                     
       Ssl hostnames:                 myapp.remote.jonatkinson.co.uk www.myapp.co
       Ssl expires at:                Aug  7 11:12:20 2020 GMT 
       Ssl issuer:                    C = US, O = Lets Encrypt, CN = Lets Encrypt Authority X3
       Ssl starts at:                 May  9 11:12:20 2020 GMT 
       Ssl subject:                   subject=CN = myapp.remote.jonatkinson.co.uk
       Ssl verified:                  self signed              
=====> myapp checks information
       Checks disabled list:          none                     
       Checks skipped list:           none                     
=====> myapp docker options information
       Docker options build:                                   
       Docker options deploy:         --restart=on-failure:10  
       Docker options run:                                     
=====> myapp domains information
       Domains app enabled:           true                     
       Domains app vhosts:            myapp.remote.jonatkinson.co.uk www.myapp.co
       Domains global enabled:        true                     
       Domains global vhosts:         remote.jonatkinson.co.uk 
=====> myapp git information
       Git deploy branch:             master                   
       Git global deploy branch:      master                   
       Git keep git dir:              false                    
       Git rev env var:               GIT_REV                  
       Git sha:                       fbe7246                  
=====> myapp network information
       Network attach post create:    
       Network attach post deploy:    
       Network bind all interfaces:   false
       Network web listeners:         172.17.0.3:5000
=====> myapp nginx information
       Nginx access log path:         /var/log/nginx/myapp-access.log
       Nginx bind address ipv4:                                
       Nginx bind address ipv6:       ::                       
       Nginx error log path:          /var/log/nginx/myapp-error.log
       Nginx hsts:                    true                     
       Nginx hsts include subdomains: true                     
       Nginx hsts max age:            15724800                 
       Nginx hsts preload:            false                    
=====> myapp proxy information
       Proxy enabled:                 true
       Proxy port map:                http:80:5000 https:443:5000
       Proxy type:                    nginx
/tmp/bashenv.982325212: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
/tmp/bashenv.180345302: line 4: export: `=/home/dokku/.basher/bash': not a valid identifier
=====> myapp ps information
       Processes:                     1                        
       Deployed:                      true                     
       Running:                       true                     
       Restore:                       true                     
       Restart policy:                on-failure:10            
       Ps can scale:                  true                     
       Status web.1:                  running    (CID: 8a2ef7d7dc36)
=====> myapp scheduler-docker-local information
       Scheduler docker local disable chown:                          
=====> myapp storage information
       Storage build mounts:                                   
       Storage deploy mounts:                                  
       Storage run mounts:                                     

### How (deb/make/rpm) and where (AWS, VirtualBox, physical, etc.) was Dokku installed?:

`bootstrap.sh`, Debian Buster, Hetzner.

**Comment by josegonzalez** [comment]
This was previously reported and resolved in #3978. Please upgrade `plugn` on your host:

```
apt update
apt install plugin=0.5.0
```

**Comment by jonatkinson** [comment]
Apologies for the dupe. Thanks for your fast response.
